modules/context/repo.go

```diff
@@ -399,6 +399,13 @@ func repoAssignment(ctx *Context, repo *repo_model.Repository) {
 	ctx.Data["PushMirrors"] = pushMirrors
 	ctx.Data["RepoName"] = ctx.Repo.Repository.Name
 	ctx.Data["IsEmptyRepo"] = ctx.Repo.Repository.IsEmpty
+
+	repoLicenses, err := repo_model.GetRepoLicenses(ctx, ctx.Repo.Repository)
+	if err != nil {
+		ctx.ServerError("GetRepoLicenses", err)
+		return
+	}
+	ctx.Data["Licenses"] = repoLicenses.StringList()
```

Comment:
I would suggest to use at least two words for variable names.

The single word names already caused various problems in history.

Reply:
Done in https://github.com/go-gitea/gitea/pull/24872/commits/b45f9073474da559895ea40487c82fd3155f955a.